core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -404,12 +410,16 @@ public static void configureHadoopConf(Object maybeConfigurable, Object conf) {
   }
 
   /**
-   * Load a custom {@link MetricsReporter} implementation.
+   * Load a custom {@link MetricsReporter} implementation. This method also supports loading
```

Comment:
I am wondering if it is cleaner to deprecate this and create a new method and property/config (`METRICS_REPORTER_IMPL_LIST`)?
```
List< MetricsReporter > loadMetricsReporters(...)
```

Reply:
I think we initially want to hide this detail and just use the existing property. From a user's perspective it shouldn't make any difference